external-import/recorded-future/src/rflib/make_markdown_table.py

```diff
@@ -11,7 +11,9 @@ def make_markdown_table(array):
 
     markdown += nl
     for entry in array[1:]:
-        markdown += f"| {' | '.join(entry)} |{nl}"
+        entry_has_list = any(isinstance(x, list) for x in entry)
+        if not entry_has_list:
```

Comment:
This looks okay to me. Should we add a log for debug purposes highlighting that one of the objects has a type that is not handled and then the whole entry will be discarded ?

Reply:
@flavienSindou Thanks :) I think this approach works for now, but the next time we need to improve or rework the connector, it would be worthwhile to consider adding it. For a quick fix, our priority is simply to ensure that parsing doesn't block the ingestion of the incident :)